django/wagtail/research/comparisons-to-existing-tools.md

```diff
@@ -0,0 +1,98 @@
+for applications that use highly normalized data. Moreover, the Django admin
+site does not offer a rich text editor. For these reasons, we've had to compile
+extensive documentation for client use of the Django admin site, with only
+limited adoption in the long term.
+
+### `feincms3`
+
+`feincms3` is an extension of the Django admin interface that's optimized for
+managing more complicated content hierarchies, e.g., nested pages. It has
+[fairly complete documentation](https://feincms3.readthedocs.io/en/latest/index.html),
+plus [a helpful example application](https://github.com/matthiask/feincms3-example),
+however the developer experience is mixed. In particular, `feincms3` [implements
+_only_ abstract classes and mixins](https://feincms3.readthedocs.io/en/latest/introduction.html),
+so standing up an instance requires a certain familiarity with these concepts,
+which isn't always true of newer developers.
+
+The user interface is similar to the Django admin interface, with the added
+ability to organize content in a tree editing interface. Unfortunately, this
+interface is slightly confusing, and it doesn't always behave as expected.
+`feincm3` also has an optional CKEditor integration, enabling rich text
+editing, but the [documentation for `django-ckeditor`](https://github.com/django-ckeditor/django-ckeditor)
+is sparse and configuration can be cryptic. Moreover, because CKEditor supports
+so many kinds of content editing, none of them offer an excellent user
+experience.
+
+Unlike Wagtail and Django CMS, `feincms3` does not offer documentation for
+content editors. For this reason, and because the user interface isn't terribly
+straightforward, we've had to write especially detailed documentation for
+clients to successfully manage content on their own.
+
+### Django CMS
+
+Wagtail and Django CMS are both built for use with Django, but they ask
+developers to employ substantially different modes of thinking. The way the two
+CMSs structure their data across views and models is quite different.
+
+Django CMS operates with plugin-based abstractions, where pages are build from
+discrete reusable units. It has some recommended default plugins, like a text
+box and image field, and the ability to build and insert custom plugins.
+
+While Django CMS's large community-built plugin library initially looked
+promising, we found that both third-party and default plugins for basic
+functions, such as rich text boxes to allow users to input and style text,
+had confusing interfaces and were often buggy.
+
+While Django CMS offers greater flexibility and encourages users to simply click
+on section of a page to change it, in practice this was overwhelming. With so
+much flexibility, plus the UI bugs outlined above, we've again needed to write
+extensive documentation for clients.
+
+### Wagtail
+
+In our experience, Wagtail is easiest to set up for developers and easier to
```

Comment:
This section is lighter than all the others! 🙀 Can you expand this along the four points outlined at the top of the discussion, @beamalsky?